### PR TITLE
Add -E flag to exit with the pr sate on unix shell

### PR DESCRIPTION
**Pull request by chmouel**
Add the -E or --exit-with-pipelinerun-error flag to tkn pr logs to exit with the pipelinerun error after showing the logs.

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

### CODE REVIEW

File: pkg/cmd/pipelinerun/logs.go [hunk 1]
```diff
@@ -109,9 +113,33 @@ func Run(opts *options.LogOptions) error {
...
 	log.NewWriter(log.LogTypePipeline, opts.Prefixing).Write(opts.Stream, logC, errC)
 
+	// get pipelinerun status
+	if opts.ExitWithPrError {
+		clients, err := opts.Params.Clients()
+		if err != nil {
+			return err
+		}
+		ctx := context.Background()
+		pr, err := clients.Tekton.TektonV1().PipelineRuns(opts.Params.Namespace()).Get(ctx, opts.PipelineRunName, metav1.GetOptions{})
```
Comment (piyush-garg) [inline pkg/cmd/pipelinerun/logs.go:123]: better to use get function from pipelinerunpkg which uses dynamic client, like you can do `pipelinerunpkg.GetPipelineRun(pipelineRunGroupResource, clients, opts.PipelineRunName, opts.Params.Namespace())`

**Comment by piyush-garg** [comment]
lgtm

**Review by vinamra28 — APPROVED** [review]
lgtm

**Pull request merged**